lib/Graph/Graph.cpp

```diff
@@ -295,6 +295,14 @@ void Module::eraseFunction(Function *F) {
   delete F;
 }
 
+uint64_t Module::dataSize() {
+  size_t size = 0;
```

Comment:
`uint64_t` again

Reply:
`getDataSize` as we also have `getSizeInBytes` and `getElementSize` in Glow's source code?